amcl/src/amcl_node.cpp

```diff
@@ -326,16 +330,19 @@ AmclNode::AmclNode() :
         odom_(NULL),
         laser_(NULL),
 	      private_nh_("~"),
-        initial_pose_hyp_(NULL),
+        initial_poses_hyp_(),
         first_map_received_(false),
         first_reconfigure_call_(true),
         tdr_("AMCL")
 {
   boost::recursive_mutex::scoped_lock l(configuration_mutex_);
 
   // Grab params off the param server
-  private_nh_.param("use_map_topic", use_map_topic_, false);
-  private_nh_.param("first_map_only", first_map_only_, false);
+  private_nh_.param("use_map_topic", use_map_topic_, true);
```

Comment:
I did not mean to change these, was fighting the values not getting updated when running the node standalone in the debugger.  Turns out it was just that a roscore was running in a separate window (param server was caching the values).